task/06-conditions-n-loops-tasks.js

```diff
@@ -30,10 +29,15 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+  return num % 15 == 0
+    ? "FizzBuzz"
+    : num % 3 == 0
+    ? "Fizz"
+    : num % 5 == 0
+    ? "Buzz"
+    : num;
```

Comment:
It's so hard to read this. It will be better if you use simple if-else statements